### PR TITLE
[13.0][FIX] purchase_request: Proper Run Buy implementation

### PR DESCRIPTION
**Pull request by HviorForgeFlow**
This fix aims to properly run either PR or PO when multiple procurements are thrown.

- [x] Stock.rule is already defined on Procrement() Object, Redo checks.

CC @Forgeflow @kongrattapong

### CODE REVIEW

File: purchase_request/__manifest__.py [hunk 1]
```diff
@@ -1,9 +1,9 @@
-# Copyright 2018-2019 Eficent Business and IT Consulting Services S.L.
+# Copyright 2018-2019 ForgeFlow, S.L.
 # License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl-3.0).
 
 {
     "name": "Purchase Request",
-    "author": "Eficent, " "Odoo Community Association (OCA)",
+    "author": "ForgeFlow, " "Odoo Community Association (OCA)",
```
Comment (MiquelRForgeFlow) [inline purchase_request/__manifest__.py]:
```suggestion
    "author": "ForgeFlow, Odoo Community Association (OCA)",
```

**Review by MiquelRForgeFlow — APPROVED** [review]
LGTM

**Review by AaronHForgeFlow — COMMENTED** [review]
Created to Orderpoints, one for directly create a PO and the other for purchase requests. Run the scheduler. Only the PO was created for the product. No purchase request was created. 

For testing I used newly created products and two different vendors for each product. One product had purchase request + buy as routes. The order had just Buy.

**Review by AaronHForgeFlow — APPROVED** [review]
:+1: Dismiss my previous comment, it works. 

off topic: it was something related to multicompany, for some reason the orderpoint was defaulted to Chicago instead of San Francico, even when I was working in San Francisco :thinking:

**Pull request merged**